bin/check-database-roles.sh

```diff
@@ -16,6 +16,8 @@ ENVIRONMENT=$2
 
 ./bin/terraform-init.sh "infra/$APP_NAME/database" "$ENVIRONMENT"
 DB_ROLE_MANAGER_FUNCTION_NAME=$(terraform -chdir="infra/$APP_NAME/database" output -raw role_manager_function_name)
+DB_CONFIG=$(terraform -chdir="infra/$APP_NAME/database" output -json db_config)
```

Comment:
you'll want to merge main into your branch. we've made all local vars lower case. see https://github.com/navapbc/template-infra/pull/629/files